spec/classes/corosync_spec.rb

```diff
@@ -308,6 +308,29 @@
       end
     end
 
+    context 'when authkey is a string' do
+      before do
+        params.merge!(
+          authkey_source: 'string',
+          authkey: 'mysecretkey'
+        )
+      end
+      it 'should deploy authkey file' do
```

Comment:
Can you fix the rubocop test failures by changing this to `it 'deploys authkey file' do` ?

Reply:
My mistake ! It's now fixed.